.github/workflows/run-test-cases.yml

```diff
@@ -84,7 +109,22 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install kubernetes
-    
+
+    - name: Build local containers for PR tests
+      if: startsWith(github.event_name, 'pull_request')
+      env:
+        BUILD_AMD64: 1
+        BUILD_ARM32: 0
```

Comment:
Should/Could we also test on arm?

Reply:
sounds good ... i don't know how to run a workflow in an arm container.  sounds like a good enhancement :)